tests/cli/workflow/mip/conftest.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+
+from cg.meta.workflow.mip import AnalysisAPI
+
+
+class MockLimsAPI:
```

Comment:
@moonso is there a good MickLimsAPI that can be reused here?
Seems like BALSAMIC is using this one as well. At least it can be moved up in the conftests.

Reply:
@henrikstranneheim AFAIK there isnt one and this one is very simple (only mocks getting target bed from LIMS and not other functionality). So I didnt dare add it in with other mocks. But we could move it there, if its good enough!

Reply:
No there is no LIMS mock yet. We need to discuss how to set up a stable solution for that here https://github.com/Clinical-Genomics/project-planning/issues/25

Reply:
👍